package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file is not needed (possible result of running `npm install` from mercator root.